contracts/vendor/proxy/EIP173Proxy.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Where do we use EIP173Proxy.sol ?

Reply:
Our custom`EIP173ProxyWithReceive` inherits `EIP173Proxy`